RxFeedback.podspec

```diff
@@ -1,48 +1,49 @@
 Pod::Spec.new do |s|
-  s.name         = "RxFeedback"
-  s.version      = "1.0.2"
-  s.summary      = "Simplest architecture for RxSwift. State + feedback loops."
-  s.description  = <<-DESC
-    Simplest architecture for RxSwift. State + feedback loops.
-    
-    * Simple
-        * If the system doesn't have state -> congrats, you have either a pure function or an observable sequence
-        * It the system does have state, here we are :)
-        * Interaction with that state is by definition a feedback loop.
-        * =>  It's just state + CQRS
-    * Straightforward
-        * if it's state -> State
-        * if it's a way to modify state -> Event/Command
-        * it it's an effect -> encode it into part of state and then design a feedback loop
-    * Declarative
-        * System behavior is first declaratively specified and effects begin after subscribe is called => Compile time proof there are no "unhandled states"
-    * Debugging is easier
-        * A lot of logic is just normal pure function that can be debugged using Xcode debugger, or just printing the commands.
-
-    * Can be applied on any level
-        * [Entire system](https://kafka.apache.org/documentation/)
-        * application (state is stored inside a database, CoreData, Firebase, Realm)
-        * view controller (state is stored inside `system` operator)
-        * inside feedback loop (another `system` operator inside feedback loop)
-    * Works awesome with dependency injection
-    * Testing
-        * Reducer is a pure function, just call it and assert results
-        * In case effects are being tested -> TestScheduler
-    * Can model circular dependencies
-    * Completely separates business logic from effects (Rx).
-        * Business logic can be transpiled between platforms (ShiftJS, C++, J2ObjC)
-  DESC
-  s.homepage     = "https://github.com/kzaher/RxFeedback"
-  s.license      = { :type => "MIT", :file => "LICENSE" }
-  s.author             = { "Krunoslav Zaher" => "krunoslav.zaher@gmail.com" }
-  s.ios.deployment_target = "8.0"
-  s.osx.deployment_target = "10.10"
-  s.watchos.deployment_target = "2.0"
-  s.tvos.deployment_target = "9.0"
-  s.source       = { :git => "https://github.com/kzaher/RxFeedback.git", :tag => s.version.to_s }
-  s.source_files  = "Sources/**/*.swift"
-  s.frameworks  = "Foundation"
-
-  s.dependency 'RxSwift', '~> 4.0'
-  s.dependency 'RxCocoa', '~> 4.0'
-end
+    s.name         = "RxFeedback"
```

Comment:
Again, no biggie but can we keep the same indentation ? (Seems like you've been using 4 spaces and the project uses 2 spaces)

Reply:
Sure @freak4pc

Reply:
I only see two spaces in my editor and is indented according to the rest.

Reply:
Look at the diffs: 
https://github.com/NoTests/RxFeedback.swift/pull/22/files

Some of the lines have extra spaces on new lines:
<img width="1083" alt="f4a5e3e6-fc99-49b5-92da-5f6d4aa99c15" src="https://user-images.githubusercontent.com/605076/35053052-cb90414e-fbb1-11e7-9881-7f744b51f179.png">

Reply:
@freak4pc I think you're missing something here, these lines are the same as the initial repository, what Github is showing you is spaces before the lines are the same. I think that spaces don't hurt anything in the structure/organization of the code isn't?

Reply:
It doesn't hurt, but if the entire project is removing trailing spaces from newlines, I wouldn't change it ... its just "garbage" characters... 

Not something I would reject for but if it was my own repo I would personally insist on as little changes as possible. Just me $0.02

Reply:
> Github is showing you is spaces before the lines are the same

It doesn't - 

On the left (original) they are deleted
On the right they are added

Reply:
@freak4pc This was missing a comma 
> Github is showing you is spaces before, the lines are the same

OK, I submitted back the files to its original state. I understand perfectly your point but these changes are regarding whitespaces and unfortunately occurred by default by XCode.